stream-chat-swift-docs/docs/guides/working-with-messages.md

```diff
@@ -1,3 +1,310 @@
 ---
 title: Working with Messages
----
\ No newline at end of file
+---
+
+## Message object
+A message is represented by `ChatMessage` model.
```

Comment:
Should this also describe the message UI object? We will probably have a separate guide for customizing the message list but some basic overview would make sense here? @b-onc ?

Reply:
There's nothing like `ChatMessageGroup` 🙈

Reply:
I mean the object you see in the message list that represents a message